tests/unit/algo/action_classification/heads/test_movinet_head.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Test of MoViNet Head."""
+
+
+import pytest
+import torch
+from otx.algo.action_classification.heads.movinet_head import MoViNetHead
+
+
+class TestMoViNetHead:
+    @pytest.fixture()
+    def fxt_movinet_head(self) -> MoViNetHead:
+        return MoViNetHead(
+            5,
+            480,
+            2048,
```

Comment:
Is there any reason to use such a high dimension in unit tests?

Reply:
Reduced by 
e904e21